censys/__init__.py

```diff
@@ -1,8 +1,7 @@
 __author__ = "Censys Team"
 __email__ = "support@censys.io"
-__copyright__ = "Copyright 2017 ZCorp Technology"
-__version__ = "0.1.0"
+__copyright__ = "Copyright 2020 Censys, Inc."
```

Comment:
idk enough about anything to know if this is ok

Reply:
I believe that the copyright is just displayed in PyPi.